### PR TITLE
LNK-2627: multi_measure_reporting_merge

### PR DESCRIPTION
**Pull request by arianamihailescu**
### 🛠 Description of Changes

Please provide a high-level overview of the changes included in this PR.

### 🧪 Testing Performed

Please describe the testing that was performed on the changes included in this PR.

### CODE REVIEW

File: DotNet/QueryDispatch/Migrations/20240909151034_AddFrequencyProperty.cs
Comment (amphillipsLGC) [inline DotNet/QueryDispatch/Migrations/20240909151034_AddFrequencyProperty.cs:1]: This doesn't seem to be doing anything?

**Review by amphillipsLGC — APPROVED** [review]
Looks good!

**Review by dvargaslantana — APPROVED** [review]

File: DotNet/QueryDispatch/Migrations/20240909151034_AddFrequencyProperty.cs
Comment (arianamihailescu) [reply DotNet/QueryDispatch/Migrations/20240909151034_AddFrequencyProperty.cs:1]: Can probably be removed. Checking with Ed since he added it.

**Comment by kissalk** [comment]
Reviewed the pipeline yaml files, and they were manually copied over - looks good

**Review by kissalk — APPROVED** [review]
Viewed devops pipeline yaml files - looks good

**Review by edward-miller-lcg — APPROVED** [review]

**Review by nvmLantana — APPROVED** [review]

**Pull request merged**